### PR TITLE
Add instructions to Feedin-Time Series menu

### LINKED ISSUE #85: Missing description in Feedin-Timeseries Tab
Add a small description what this tab is all about. Mention that there are currently only test-data.

### PR DESCRIPTION
**Pull request by jh-RLI**
closes #85

### CODE REVIEW

**Comment by jh-RLI** [comment]
closes #74

File: templates/WAM_APP_FRED/test_map_layout.html [hunk 1]
```diff
@@ -176,7 +176,11 @@ <h4>Weather Data</h4>
 
                   <!-- Third panel content -->
                   <div class="tabs-panel" id="panel-feedinlib">
-                    test
+                    <p>
+                      Click on a region displayed on the map. Click the now visible datapoint displayed
```
Comment (Bachibouzouk) [inline templates/WAM_APP_FRED/test_map_layout.html]:
```suggestion
                      Select a variable and a time span, then click on a region highlighted on the map. The data will be displayed on a graph inside a popup.
```

The popup should open automatically : #91

File: templates/WAM_APP_FRED/test_map_layout.html [hunk 2]
```diff
@@ -176,7 +176,11 @@ <h4>Weather Data</h4>
 
                   <!-- Third panel content -->
                   <div class="tabs-panel" id="panel-feedinlib">
-                    test
+                    <p>
+                      Click on a region displayed on the map. Click the now visible datapoint displayed 
+                      in the middle of the specific region. Currently the plot in the popup window
```
Comment (Bachibouzouk) [inline templates/WAM_APP_FRED/test_map_layout.html]:
```suggestion
                       Currently the plot in the popup window 
```

**Review by Bachibouzouk — APPROVED** [review]
Thank :)

**Pull request merged**